src/__tests__/verifyKey.ts

```diff
@@ -0,0 +1,39 @@
+import { verifyKey } from '../index';
+import {
+  generatedInvalidKeyPair,
+  generatedValidKeyPair,
+  pingRequestBody,
+  signRequestWithKeyPair,
+} from './utils/SharedTestUtils';
+
+describe('verify key method', () => {
+  it('valid', () => {
+    // Sign and verify a valid request
+    const signedRequest = signRequestWithKeyPair(pingRequestBody, generatedValidKeyPair.secretKey);
```

Comment:
Not a blocker, but perhaps you want to parameterize these tests to also try `applicationCommandRequestBody`?